daisy_integration_tests/scripts/post_translate_test.sh

```diff
@@ -145,17 +145,13 @@ function check_cloud_init {
 function check_package_install {
   # Apt
   if [[ -d /etc/apt ]]; then
-    status "Checking if apt can update cache."
-    apt-get update
-    if [[ $? -ne 0 ]]; then
-      fail "apt-get update failed."
-    fi
-
     status "Checking if apt can install a package."
-    apt-get install --reinstall make
-    if [[ $? -ne 0 ]]; then
-      fail "apt-get cannot install make."
-    fi
+    for i in $(seq 1 20) ; do
+      apt-get update && apt-get install --reinstall make && return 0
```

Comment:
why are these two checks merged?

Reply:
The previous code was making two assertions -- one that apt was updatable and one that make could be installed -- and either assertion could fail if the autoupdate script was holding a dpkg lock. By chaining them in this way, we benefit from one set of retry logic that will wrap both.